packages/react/src/components/Masthead/Masthead.js

```diff
@@ -103,8 +107,27 @@ const Masthead = ({ navigation, ...mastheadProps }) => {
         );
       }
     });
+
```

Comment:
@annawen1 as discussed, we should move this out into an `AnalyticsAPI`, which will have the `registerEvent` method, as well as a `initScrollTracker` method.

Reply:
done